### PR TITLE
app-emulation/punes: new package

### PR DESCRIPTION
**Pull request by winterheart**
puNES is Nintendo Entertainment System emulator.

Package-Manager: Portage-2.3.13, Repoman-2.3.3

### CODE REVIEW

File: app-emulation/punes/punes-0.102.ebuild [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+S="${WORKDIR}/puNES-${PV}"
+
+src_prepare() {
+	default
+
+	sed -i "/update-desktop-database/d" misc/Makefile.am || die
+	eautoreconf
+	# FIXME why eautoreconf can't handle this?
+	cd "src/extra/lib7zip-1.6.5"
```
Comment (Zlogene) [inline app-emulation/punes/punes-0.102.ebuild]: `|| die`

**Comment by winterheart** [comment]
@Zlogene done

File: app-emulation/punes/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="person">
+<email>azamat.hackimov@gmail.com</email>
```
Comment (Zlogene) [inline app-emulation/punes/metadata.xml]: should also states gentoo proxy-maint team here, as you are not a developer.

Comment (winterheart) [reply app-emulation/punes/metadata.xml]: @Zlogene done.

**Comment by monsieurp** [comment]
Merging underway.

**Pull request closed without merging**